runtime/common/BaseRestRemoteClient.h

```diff
@@ -913,6 +913,7 @@ class BaseNvcfRuntimeClient : public cudaq::BaseRemoteRestRuntimeClient {
         (funcEnv.majorVersion >= 1 && funcEnv.minorVersion >= 1);
     capabilities.vqe = funcEnv.majorVersion > 1 ||
                        (funcEnv.majorVersion >= 1 && funcEnv.minorVersion >= 1);
+    capabilities.isRemoteSimulator = true;
```

Comment:
@bmhowe23 added a new capability so we can pass it to argument conversion during synthesis.